gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip
```

Comment:
while I like updating things I don't think updating it in a "maybe unrelated" PR is that clear. Could you please elaborate on the why?

Reply:
Yeah thanks for keeping me honest, I'll pull this into its own PR.